clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/securitygroup/SecurityGroupLookupFactory.groovy

```diff
@@ -136,9 +136,12 @@ class SecurityGroupLookupFactory {
          */
         retrySupport.retry({
           CreateTagsRequest createTagRequest = new CreateTagsRequest()
-          createTagRequest.withResources(result.groupId).withTags([
-            new Tag("Name", description.name)
-          ])
+          Collection<Tag> tags = new HashSet()
```

Comment:
This should just be upserting the nobu tags right?

Reply:
yes , that is correct.